### PR TITLE
add devcontainer json

### PR DESCRIPTION
**Pull request by franklinlindemberg**
## Purpose

* Adds .devcontainer file
* Updates README with UI Deploy
 
## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
Documentation

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,6 @@
-# AI-in-a-Box Accelerator Name
-
-<!-- <div style="display: flex;">
```
Comment (franklinlindemberg) [inline README.md:3]: should we remove this from the aib template as well?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
```
Comment (franklinlindemberg) [inline .gitattributes:1]: Im curious what this do?

**Review by franklinlindemberg — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (codebytes) [reply README.md:3]: The template has this as a placeholder, to replace in the new project.

File: .gitattributes (hunk 2 above)
Comment (codebytes) [reply .gitattributes:1]: for dev containers, dealing with line endings from windows to linux.
https://code.visualstudio.com/docs/devcontainers/tips-and-tricks#_resolving-git-line-ending-issues-in-containers-resulting-in-many-modified-files

**Pull request merged**